package/osx/patch-wxwidgets.diff

```diff
@@ -0,0 +1,46 @@
+diff --git src/osx/carbon/dataobj.cpp src/osx/carbon/dataobj.cpp
```

Comment:
Why is patching wxWidgets necessary?

Reply:
Because wxWidgets 3.0.2 doesn't compile on recent macOS. It includes QuickTime.h which is not present anymore. Another option would be to make Alien::wxWidgets download 3.0.4 but I think that's more work.